<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fAddManager</class>
 <widget class="QDialog" name="fAddManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>410</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>931</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbName">
      <property name="text">
       <string>Имя менеджера</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbAddress">
      <property name="text">
       <string>Адрес</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leAddress"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbPassport">
      <property name="text">
       <string>Паспорт</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lePassport"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbPhone">
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lePhone"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbBirthday">
      <property name="text">
       <string>День рождения</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="leBirthday"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbInn">
      <property name="text">
       <string>ИНН</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="leInn"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lbEmpDate">
      <property name="text">
       <string>Дата контракта</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="leDateEmp"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lbPercent">
      <property name="text">
       <string>Процент</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lePercent"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="lbEmail">
      <property name="text">
       <string>Почта</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="leEmail"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="lbNumOfGroup">
      <property name="text">
       <string>Номер группы</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="leNumOfGroup"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbCaption">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавление менеджера</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>fAddManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>fAddManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
